listing-description/en-US.md

```diff
@@ -2,39 +2,35 @@
 
 A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[![Video review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```

Comment:
On modrinth and on the CF description, this should likely be replaced by

```html
<iframe allowfullscreen="allowfullscreen" src="https://www.youtube.com/embed/bb8G9X5Q_4I" height="358" width="638">
</iframe>
```

Reply:
Nah. It used to be an embed before, but an embed is not scalable (in the listings) for mobile and even previewing it wastes computer resources, hence thumbnails were chosen. Since the upcoming website will display one language at once and is better scalable, there an actual embed could be used.